### PR TITLE
Fix the clustershell roster host names

### PR DESCRIPTION
**Pull request by matdrapeau**
This fixes: https://github.com/saltstack/salt/issues/22669

### CODE REVIEW

**Comment by thatch45** [comment]
We still need to get a virtual function on this module, it is likely generating errors in the logs for those without clusterssh installed

File: salt/roster/clustershell.py [hunk 1]
```diff
@@ -14,9 +14,17 @@
 # Import python libs
 from __future__ import absolute_import
 import socket
-
 from salt.ext.six.moves import map  # pylint: disable=import-error,redefined-builtin
-from ClusterShell.NodeSet import NodeSet
+
```
Comment (matdrapeau) [inline salt/roster/clustershell.py]: @thatch45 let me know if this is what you expect?

File: salt/roster/clustershell.py [hunk 2]
```diff
@@ -14,9 +14,17 @@
...
+REQ_ERROR = None
+try:
+    from ClusterShell.NodeSet import NodeSet
+except (ImportError, OSError) as e:
+    REQ_ERROR = 'ClusterShell import error, perhaps missing python ClusterShell package'
+
+
+def __virtual__():
+    return (REQ_ERROR is None, REQ_ERROR)
```
Comment (matdrapeau) [inline salt/roster/clustershell.py]: @thatch45 let me know if this is what you expect?

Comment (thatch45) [inline salt/roster/clustershell.py]: Perfect! Awesome!

**Comment by jfindlay** [comment]
@matdrapeau, it looks like you've accidentally included commits that aren't intended to be part of your pull request.  Is it possible for you to resubmit such that they aren't included?  Thanks.

**Comment by matdrapeau** [comment]
Clean version pushed...

**Comment by jfindlay** [comment]
Nice, thanks @matdrapeau.

**Pull request closed without merging**